### DISCUSSION TITLE
Use parent inscription's variables in child inscription

### DISCUSSION THREAD

**Discussion by SatoshiRoppongi** [ordinals/ord#2190] [Q&A]
Is there a way to retain variables from one inscription and use them in a referenced inscription?

For example, is there a way to have an "2" in an inscription, increment it to show a "3" in a recursive inscription, and then show a "4" in a recursive inscription?

**Comment by The-Inscriptor** [comment]
This would be super useful. Also a way to query block-height (and mint date) of parent inscription by its children (or vice versa).

**Reply by SatoshiRoppongi** [reply]
Thank you for your response.
I mean, 
if parent inscription (javascript) has a viriable "a" and it's 1 (for example).
is it possible to refer "a" from other inscriptions like

b = refer("content/${parentsInscriptionsVariableA}") + 1;

(this might no be a right grammer)
